source/MaterialXView/Viewer.cpp

```diff
@@ -935,8 +935,6 @@ void Viewer::createAdvancedSettings(Widget* parent)
     referenceQualityBox->setCallback([this](bool enable)
     {
         _genContext.getOptions().hwDirectionalAlbedoMethod = enable ? mx::DIRECTIONAL_ALBEDO_IS : mx::DIRECTIONAL_ALBEDO_TABLE;
-        // No Albedo Table support for Essl yet.
```

Comment:
I don't quite understand why this is being removed.
I did see a number of places which does not perform and conditional #ifdef build 
which includes these places.

Reply:
They were not in #ifdef, wasn't sure if _genContextEssl needs to be configured for UI updates. I've added ifdefs for _genContextEssl now